frontend/src/global_styles/content/_table.sass

```diff
@@ -234,10 +234,15 @@ thead.-sticky th
   height:  var(--generic-table--header-height)
   z-index:      1
   border-bottom: 1px solid var(--table-row-border-color)
+  display: flex
 
   &:hover,
   &.hover
     background:   #f8f8f8
+  
+  & > op-icon
+    position: absolute
```

Comment:
Absolute positioning is somehow overkill in this situation as the elements are all displayed inline and you even added a flex box to the parent.